hwi.spec

```diff
@@ -1,21 +1,29 @@
 # -*- mode: python -*-
 import platform
 import subprocess
+import os
 
 block_cipher = None
 
 binaries = []
+
 if platform.system() == 'Windows':
-    binaries = [("c:/python3/libusb-1.0.dll", ".")]
+    libusb_path = 'c:/python3/libusb-1.0.dll'
 elif platform.system() == 'Linux':
-    binaries = [("/lib/x86_64-linux-gnu/libusb-1.0.so.0", ".")]
+    if os.path.exists('/lib/x86_64-linux-gnu/libusb-1.0.so.0'):
+        libusb_path = '/lib/x86_64-linux-gnu/libusb-1.0.so.0'
+    elif os.path.exists('/usr/lib/libusb-1.0.so.0'):
+        libusb_path = '/usr/lib/libusb-1.0.so.0'
+    else:
+        libusb_path = '/lib/libusb-1.0.so.0'
```

Comment:
I wonder if there is some way to automatically discover the path to libusb similar to what we do for Mac? Maybe something that uses ld?

Reply:
This seems relevant: https://github.com/albertz/system-tools/blob/master/bin/find-lib-in-path.py

Reply:
I can parse `/etc/ld.so.conf` like the code @prusnak referenced, but on my system it doesn't help and it adds some complexity. I also can't copy paste the it right now since it doesn't have a license.
Also note that it doesn't address the priority between these paths, and I'm not aware of a way to do it.

What do you think is best here?
Should I parse `/etc/ld.so.conf`, or keep it as is?

Reply:
You could at least try to refactor the code to something like this so it is more extensible in the future:

``` python
libusb_path = None
paths = ["/lib/x86_64-linux-gnu", "/usr/lib/x86_64-linux-gnu", "/lib64", "/usr/lib64", "/lib", "/usr/lib"]
for path in paths:
    lib = path + "/libusb-1.0.so.0" 
    if os.path.exists(lib):
        libusb_path = lib
        break
```

Reply:
Done

Reply:
I was thinking something more like run `ldconfig -p | grep "libusb-1.0.so.0 (libc6,x86-64)"` and take the path from that.

Reply:
That relies on glibc and will break on musl (so alpine and others won't work)